### DISCUSSION TITLE
Package can't be installed: "System command 'R' failed"

### DISCUSSION THREAD

**Discussion by emmapaeronpuss** [USGS-R/streamMetabolizer#428] [Q&A] [answered]
Hi there! 
I've been trying to install the StreamMetabolizer package for a while now and don't know how to do it, as I always get this error:
"SUMMARY: processing the following file failed:
     ‘data_prep.Rmd’
Error: Vignette re-building failed.
   Execution halted
Error: Failed to install 'streamMetabolizer' from GitHub:
  ! System command 'R' failed". 

Those are the steps I've tried to do before, as suggested by reddit forums and chatgpt:
system("xcode-select --install")
install.packages("StanHeaders", type = "source")
install.packages("rstan", repos = "https://cloud.r-project.org/", dependencies = TRUE)
install.packages("rlang")

and then tried: 
remove.packages(c("rstan","StanHeaders"))
install.packages("StanHeaders", repos = c("https://mc-stan.org/r-packages/", getOption("repos")))
install.packages("rstan", repos = c("https://mc-stan.org/r-packages/", getOption("repos")))

And then to install StreamMetabolizer:
install.packages('remotes')
remotes::install_github('appling/unitted')
remotes::install_github("DOI-USGS/streamMetabolizer", build_vignettes = TRUE)

I have the R Version 2023.12.0+369.

An idea? Thank youuuuu

**Comment by aappling-usgs** [comment, marked as answer]
Try this:
```r
remotes::install_github("DOI-USGS/streamMetabolizer", build_vignettes = FALSE)
```

(once that works, we can look deeper at the vignette-building issue)

**Reply by emmapaeronpuss** [reply]
I did this and installed the newest version of RStudio, it worked! Thank you!
